docs/protocols/a2a-guide.mdx

```diff
@@ -816,9 +816,10 @@ const response = await a2a.send({
   }
 });
 
-// Check for file validation issues
-if (response.status === 'failed' && response.data?.file_errors) {
-  console.log('File issues:', response.data.file_errors);
+// Check for file validation issues (extract from A2A artifact structure)
+const dataPart = response.artifacts?.[0]?.parts?.find(p => p.kind === 'data');
```

Comment:
one - this is kinda gross syntax no? can't be the right way to find this. but more importantly, this is why we have testable snippets. let's convert all of these to testable snippets so they "just work"

Reply:
updated!!